aws-ssm-maintenancewindowtarget/src/main/java/software/amazon/ssm/maintenancewindowtarget/translator/property/PropertyTranslator.java

```diff
@@ -0,0 +1,27 @@
+package software.amazon.ssm.maintenancewindowtarget.translator.property;
```

Comment:
Just a question, should we do this for MW and MWTask resource as well?

Reply:
I think it depends which is better in terms of organization. I chose to implement it this way so each translator can used across resources, in the event that we decide to reuse the code or merge translators in the future. On the other hand, it would make more intuitive sense to group translators by request and response, which I believe is done in MW and MWTask. It should be fine to leave it as is for now